README.md

```diff
@@ -1,2 +1,17 @@
 # breakfast-serializer
-A (hopefully) backwards compatible replacement for the JMS serializer
+
+[![Build Status](https://travis-ci.org/brettminnie/breakfast-serializer.svg)](https://travis-ci.org/brettminnie/breakfast-serializer)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/brettminnie/breakfast-serializer/badges/quality-score.png?b=develop)](https://scrutinizer-ci.com/g/brettminnie/breakfast-serializer/?branch=develop)
+[![Coverage Status](https://coveralls.io/repos/brettminnie/breakfast-serializer/badge.svg)](https://coveralls.io/r/brettminnie/breakfast-serializer)
+
+A (hopefully) backwards compatible replacement for the other well known serializer
+
+Currently will only support json and do a full depth/breadth recursion, we'll build on this slowly
+
+Example
+```php
```

Comment:
haha, in the tests

it's a serializer, documentation standards have been set :)

noted